Automattic/jetpack
# Instant Search: Update prototype with feature branch changes

#### Changes proposed in this Pull Request:
This PR updates the Instant Search prototype with the following changes:
* Bugfixes and code optimizations
* Accessibility improvements
* Network connectivity improvements
* Product improvements via design feedback
* Photon support

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Yes, this is part of the Jetpack Instant Search project: p10gg3-9iP-p2. It has not yet been released to the public.

#### Testing instructions:
1. If you haven't already, set up Instant Search following the instructions in the [readme](https://github.com/Automattic/jetpack/blob/instant-search-master/modules/search/instant-search/README.md#testing-instructions).
2. Enter a search query into your site's search input. Ensure that the search results render onto your site's main content area.

#### Proposed changelog entry for your changes:
* Bugfixes and code optimizations for Jetpack Instant Search.
